src/NamedTupleUtilities.jl

```diff
@@ -0,0 +1,136 @@
+"""
```

Comment:
Could we still leave this in a module? Just wrap everything in here in ``module NamedTupleUtilities`` and an ``end`` at the end of the file.